test/e2e/index.test.ts

```diff
@@ -1,5 +1,6 @@
 import { expect } from 'chai';
 import * as playwright from 'playwright';
+import '../utils/initPlaywrightMatchers';
```

Comment:
We use relative imports for in-module imports. For example, in core we import `'../Button'` not `'@material-ui/core/Button'`.

Reply:
Regarding the why. Is it accurate that we only use in-module import in the core because we have to publish the package on npm? (that it would break if we didn't)

Reply:
Doesn't matter if it's published on npm or not. Packages exist in node as well and imports are standardized in ES as well now. Using this absolute is just another thing one (computer or human) as to lookup and it's generally just not that important. Let's concentrate on the meat of the PR not subjective nits.

Reply:
Agree, it's not directly relevant to the changes 👍 .

I guess more a general discussion for the future, not very relevant for this one-time instance.
So we could use absolute imports everywhere? I didn't know that, ok. Would it work if there are multiple times the package in the bundle? Will we risk importing from the wrong version of the package?

I'm asking because relative imports seem to make refactoring harder (need to change the import path if we move the folder) as well as seems to make the location resolution harder compared to absolute (developers have to know where the file is located **and** where the destination is, relative to it).